src/Provider/Here/Tests/IntegrationTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Geocoder package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license    MIT License
+ */
+
+namespace Geocoder\Provider\Here\Tests;
+
+use Geocoder\IntegrationTest\ProviderIntegrationTest;
+use Geocoder\Provider\Here\Here;
+use Http\Client\HttpClient;
+
+/**
+ * @author Sébastien Barré <sebastien@sheub.eu>
```

Comment:
In the `Here` class you annotate the class with [info@zoestha.de](https://github.com/geocoder-php/Geocoder/pull/844/files#diff-fa50f87c5dfdc2fde6b82c55626dce1aR28) and here with `sebastien@sheub.eu`. Is this intended or do you want to fix this?

Reply:
Thanks for the Review. I have corrected the Here Class.